packages/shared/src/components/modals/common/ModalFooter.tsx

```diff
@@ -1,12 +1,30 @@
-import React, { ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+import React, { ReactElement, ReactNode, useContext } from 'react';
+import { ModalPropsContext } from './types';
 
 export type ModalFooterProps = {
   children?: ReactNode;
+  className?: string;
+  justify?: 'end' | 'center' | 'between';
+  tab?: string;
 };
 
-export function ModalFooter({ children }: ModalFooterProps): ReactElement {
+export function ModalFooter({
+  children,
+  className,
+  justify = 'end',
+  tab,
+}: ModalFooterProps): ReactElement {
+  const { activeTab } = useContext(ModalPropsContext);
+  if (tab && tab !== activeTab) return null;
```

Comment:
The footer is not part of the tab right, why do we hide it?

Reply:
No, it can be, for example for the new comment modal, footer changes based on the tab.

Reply:
https://www.sketch.com/s/2867ec63-0966-4647-b1d7-b6fe696ba251/a/agogLmZ

Reply:
How would it be set for modal with tabs that have 1 generic footer?

Reply:
If you dont pass a tab, it will render it

```tsx
<Footer> Generic footer that renders for all tabs</Footer>
```

```tsx
<Footer tab="Preview">Special footer that renders for only if the Preview tab is active</Footer>
```